teams/teams-ps/teams/Set-CsTeamsCallingPolicy.md

```diff
@@ -484,6 +486,38 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -PayAsYouGoSpendingLimits
+Determines if spending limit is enabled for pay-as-you-go PSTN calls.
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Aliases:
+Applicable: Microsoft Teams
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PayAsYouGoSpendingUserLimit
+Determines the value of spending limit for a user making pay-as-you-go PSTN calls.
```

Comment:
Please add details on what the possible values are

Reply:
fixed